gms/impl/src/main/java/com/linkedin/metadata/resources/entity/IngestionRunResource.java

```diff
@@ -0,0 +1,130 @@
+package com.linkedin.metadata.resources.entity;
+
+import com.linkedin.metadata.aspect.VersionedAspect;
+import com.linkedin.metadata.entity.EntityService;
+import com.linkedin.metadata.entity.ebean.EbeanEntityService;
+import com.linkedin.metadata.restli.RestliUtils;
+import com.linkedin.metadata.run.AspectRowSummary;
+import com.linkedin.metadata.run.AspectRowSummaryArray;
+import com.linkedin.metadata.run.IngestionRunSummaryArray;
+import com.linkedin.metadata.run.RollbackResponse;
+import com.linkedin.metadata.systemmetadata.SystemMetadataService;
+import com.linkedin.parseq.Task;
+import com.linkedin.restli.server.annotations.Action;
+import com.linkedin.restli.server.annotations.ActionParam;
+import com.linkedin.restli.server.annotations.Optional;
+import com.linkedin.restli.server.annotations.RestLiCollection;
+import com.linkedin.restli.server.resources.CollectionResourceTaskTemplate;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import javax.inject.Named;
+import lombok.extern.slf4j.Slf4j;
+
+
+/**
+ * resource for showing information and rolling back runs
+ */
+@Slf4j
+@RestLiCollection(name = "runs", namespace = "com.linkedin.entity")
+public class IngestionRunResource extends CollectionResourceTaskTemplate<String, VersionedAspect> {
```

Comment:
We should try to differentiate between 2 forms of ingestion throughout our abstractions:

Case 1: Batch Ingestion - This is what the metadata ingestion framework is currently limited to
Case 2: Stream Ingestion - True push ingestion. 

"IngestionRun" only makes sense in the batch case. Let's think about renaming this accordingly

Reply:
updated to BatchIngestionRunResource